CMakeLists.txt

```diff
@@ -294,6 +294,10 @@ push/InfluxPush.cpp
 push/WebsocketPush.cpp
 httpclient/HTTPClient.cpp
 httpclient/UrlEncode.cpp
+hardware/eHouse/EhouseEvents.cpp
+hardware/eHouseTCP.cpp
+hardware/eHouse/EhouseUdpListener.cpp
+hardware/eHouse/EhouseTcpClient.cpp
 hardware/1Wire.cpp
```

Comment:
Could you do a slight change in order like the 1Wire files:
+hardware/eHouseTCP.cpp
+hardware/eHouse/EhouseEvents.cpp
+hardware/eHouse/EhouseTcpClient.cpp
+hardware/eHouse/EhouseUdpListener.cpp

We try to keep the files in alphabetic order when possible, so on slow compile systems you know a bit how long it takes

Reply:
Hi, Thanks for infos and suggestion: 
I've made this change: but can I leave eHouse on the top of list?
 I do lot of compiles every day until it will be stabilized and it is much fastester.